package/src/constructs/query-param.ts

```diff
@@ -0,0 +1,2 @@
+import KeyValuePair from './key-value-pair'
+export type QueryParam = KeyValuePair
```

Comment:
Should we export these types from constructs/index.ts to make them public?

Reply:
Oh, good catch. I forgot to add them there